js/src/shortcode-ui.js

```diff
@@ -30,7 +30,8 @@ $(document).ready(function(){
 			options = {
 				frame: 'post',
 				state: 'shortcode-ui',
-				title: shortcodeUIData.strings.media_frame_title
+				title: shortcodeUIData.strings.media_frame_title,
+				editor: this.dataset.editor
```

Comment:
Just out of curiosity, why do you use `this.dataset.editor` down here, and `$(event.currentTarget).data('editor')` above? Is there any case where the two would be different?